app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,65 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    comment = resource.comments.build(comment_params.merge(user: current_user))
```

Comment:
It might be clearer to a reader to do something like
```
commentable = load_commentable # or similar
commentable.comments.build
```

resource is an overloaded term in these things.